src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import BooksList from './components/BooksList';
+import Categories from './components/Categories';
+
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/prefer-stateless-function */
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div>
+        <h1> Book store app</h1>
+        <Header />
+        <Routes>
+          <Route path="/" element={<BooksList />} />
+          <Route path="/BooksList" element={<BooksList />} />
```

Comment:
- [ ] Please, consider that, as it is required in the project, here you need to use `exact` for the BookList path to not show a blank page when visiting the site for the first time, the BookList component should be the default page. So here the lines 21 and 22 could be replaced with one:
`<Route exact path="/" element={<BooksList />} />`